lib/adapters/fs.js

```diff
@@ -14,7 +15,7 @@ module.exports = objectAssign({}, FS, {
 
   writeFile: fs.writeFile,
 
-  mkdir: fs.mkdir,
+  mkdir: mkdirp,
```

Comment:
That's Mugshot's business. Your business is providing a temporary dir to store the results in. You can create a temp dir and set the `rootDirectory` option to a non-existent folder inside the temp one. You can then remove the entire temp dir at the end of the tests.